java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -124,42 +124,6 @@
     static final ResourceBundle rbx = ResourceBundle.getBundle("jmri.jmrit.display.DisplayBundle");
     static final ResourceBundle rbean = ResourceBundle.getBundle("jmri.NamedBeanBundle");
 
-    // size of point boxes
-    private static final double SIZE = 3.0;
-    private static final double SIZE2 = SIZE * 2.;  // must be twice SIZE
-
-    // connection types
-    final public static int NONE = 0;
```

Comment:
These constants all must be deprecated, but not removed. There are external applications that embed JMRI as a library (CATS being the most obvious, but not the only one), and we do our best not to make immediately breaking changes to those applications by deprecating (using the @Deprecated annotation with a JavaDoc comment documenting in which JMRI version the deprecation happened and what the replacement is.